crates/re_data_ui/src/item_ui.rs

```diff
@@ -225,6 +225,7 @@ pub fn data_blueprint_group_button_to(
             &re_ui::icons::CONTAINER,
             text,
             ctx.selection().contains_item(&item),
+            false,
```

Comment:
Better to use an enum

```suggestion
            re_ui::LabelStyle::Named,
```